setup.sh

```diff
@@ -211,7 +211,7 @@ run() {
 
     export ROFI_BUILD_CONFIGURATION
     export ROFI_ROOT=$(pwd)
-    export ROFI_BUILD_DIR="$(pwd)/build.${ROFI_BUILD_CONFIGURATION}"
+    export ROFI_BUILD_DIR="$(pwd)/build.${ROFI_BUILD_PREFIX}${ROFI_BUILD_CONFIGURATION}"
```

Comment:
Should be documented

Reply:
Addresed in [62a523d](https://github.com/paradise-fi/RoFI/pull/171/commits/62a523d6f648ae03552427848f6e45d2b995f568)